### PR TITLE
[Feature] Baseline implementation of Vue UI framework

### PR DESCRIPTION
**Pull request by labbomb**
### Purpose

Please refer to the main `issue`  [#35](https://github.com/apache/incubator-paimon-webui/issues/35).

### CODE REVIEW

File: paimon-web-ui-new/src/config/index.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/**
+ * # Msi 全局配置
+ * @author labbomb
+ */
```
Comment (songjianet) [inline paimon-web-ui-new/src/config/index.ts:4]: delete this.

File: paimon-web-ui-new/src/config/index.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+export class Setting {
+  // #region 全局配置
+  /**
+   * # Msi版本号
+   */
```
Comment (songjianet) [inline paimon-web-ui-new/src/config/index.ts:9]: delete this.

File: paimon-web-ui-new/src/main.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+const pinia = createPinia()
+app.use(pinia)
+pinia.use(piniaPluginPersistedstate)
+app.use(router)
+app.use(naive)
+
+app.mount('#app')
+
+
+// 项目信息全局console
```
Comment (songjianet) [inline paimon-web-ui-new/src/main.ts:22]: delete this.

File: paimon-web-ui-new/src/main.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+console.clear();
+
+// 家人们谁懂啊, \/\/\/ 天依蓝、镜音连、巡音粉、初音绿 /\/\/
```
Comment (songjianet) [inline paimon-web-ui-new/src/main.ts:25]: delete this.

File: paimon-web-ui-new/vite.config.ts [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+            'useDialog',
+            'useMessage',
+            'useNotification',
+            'useLoadingBar',
+          ],
+        },
+      ],
+      dts: './auto-imports.d.ts',
+      eslintrc: {
+        enabled: false, // 1、true时生成eslint配置文件，2、生成后改为false，避免重复消耗
```
Comment (songjianet) [inline paimon-web-ui-new/vite.config.ts:31]: use english

**Review by songjianet — CHANGES_REQUESTED** [review]

**Review by songjianet — APPROVED** [review]

File: paimon-web-ui-new/.eslintrc.cjs [hunk 6]
```diff
@@ -0,0 +1,16 @@
+/* eslint-env node */
```
Comment (SteNicholas) [inline paimon-web-ui-new/.eslintrc.cjs:1]: Adds license for all cjs file.

Comment (SteNicholas) [inline paimon-web-ui-new/.eslintrc.cjs:1]: Adds license for all cjs file. You could refer to [.eslintrc.cjs](https://github.com/apache/skywalking-booster-ui/blob/main/.eslintrc.cjs) of SkyWalking.

File: paimon-web-ui-new/.gitignore [hunk 7]
```diff
@@ -0,0 +1,28 @@
+# Logs
```
Comment (SteNicholas) [inline paimon-web-ui-new/.gitignore:1]: Could this .gitignore merge with the .gitignore in parent directory?

File: paimon-web-ui-new/.prettierrc.json [hunk 8]
```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://json.schemastore.org/prettierrc",
+  "semi": false,
+  "tabWidth": 2,
+  "singleQuote": true,
+  "printWidth": 100,
+  "trailingComma": "none"
+}
```
Comment (SteNicholas) [inline paimon-web-ui-new/.prettierrc.json:8]: Add the new line at the end of this file.

File: paimon-web-ui-new/README.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
+# paimon-web-ui-new
+
+This template should help get you started developing with Vue 3 in Vite.
```
Comment (SteNicholas) [inline paimon-web-ui-new/README.md:3]:
```suggestion
This UI starts from Vue 3 in Vite.
```

File: paimon-web-ui-new/README.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+
+## Recommended IDE Setup
```
Comment (SteNicholas) [inline paimon-web-ui-new/README.md:5]:
```suggestion
## IDE Setup
```

File: paimon-web-ui-new/README.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+
+[VSCode](https://code.visualstudio.com/) + [Volar](https://marketplace.visualstudio.com/items?itemName=Vue.volar) (and disable Vetur) + [TypeScript Vue Plugin (Volar)](https://marketplace.visualstudio.com/items?itemName=Vue.vscode-typescript-vue-plugin).
+
+## Type Support for `.vue` Imports in TS
+
+TypeScript cannot handle type information for `.vue` imports by default, so we replace the `tsc` CLI with `vue-tsc` for type checking. In editors, we need [TypeScript Vue Plugin (Volar)](https://marketplace.visualstudio.com/items?itemName=Vue.vscode-typescript-vue-plugin) to make the TypeScript language service aware of `.vue` types.
+
+If the standalone TypeScript plugin doesn't feel fast enough to you, Volar has also implemented a [Take Over Mode](https://github.com/johnsoncodehk/volar/discussions/471#discussioncomment-1361669) that is more performant. You can enable it by the following steps:
+
+1. Disable the built-in TypeScript Extension
```
Comment (SteNicholas) [inline paimon-web-ui-new/README.md:15]:
```suggestion
1. Disables the built-in TypeScript extension.
```

File: paimon-web-ui-new/README.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+    1) Run `Extensions: Show Built-in Extensions` from VSCode's command palette
+    2) Find `TypeScript and JavaScript Language Features`, right click and select `Disable (Workspace)`
```
Comment (SteNicholas) [inline paimon-web-ui-new/README.md:17]:
```suggestion
    2. Finds `TypeScript and JavaScript Language Features`, right click and select `Disable (Workspace)`.
```

File: paimon-web-ui-new/README.md [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+
+```sh
+pnpm build
+```
+
+### Lint with [ESLint](https://eslint.org/)
+
+```sh
+pnpm lint
+```
```
Comment (SteNicholas) [inline paimon-web-ui-new/README.md:46]: Add license detail like [README.md](https://github.com/apache/skywalking-booster-ui/blob/56a02293cfd3b62eb46d749895ff628f1830b116/README.md).

File: paimon-web-ui-new/src/main.ts [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+console.log(
+  `%c© Msi%c(Integration)\n%c${Setting.version}`,
+  `padding:24px 0px 12px 0px;
+  font-size:24px;
+  color:#66CCFF;`,
+  'font-size:24px;color:#FFE212;',
+  'font-size:12px;color:#FFBFCB;padding-bottom:24px;',
+);
+
+console.group('%cConsole', 'font-size:12px;color:#39C5BB;font-weight:300;');
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/main.ts:49]: Ditto.

**Review by SteNicholas — APPROVED** [review]
LGTM. @labbomb, thanks for updates.

**Pull request merged**